views/proposal/create-proposal.php

```diff
@@ -12,13 +12,22 @@
 <?php
     $form = yii\widgets\ActiveForm::begin([
         'id' => 'proposalForm',
+        'options' => [
+            'enctype' => 'multipart/form-data',
+        ],
     ]);
 ?>
 <?= $form->field($model, 'title')->textInput(['id' => 'proposalFormTitleInput']); ?>
 <?= $form->field($model, 'content')->hiddenInput(['id' => 'proposalFormContentInput']); ?>
 <div id="Proposalcontent" class="editSection"></div>
-<?= $form->field($model, 'relatedFile')->fileInput(); ?>
-<?= yii\helpers\Html::submitButton('Submit'); ?>
+<div class="form-group mt-4 file-input-container">
+    <label class="file-input-label">
+        Pick or drop a file...
```

Comment:
It seems to familiar for me. Maybe you can put "Select a file" or "Choose a file"

Reply:
I think it's good not to be too formal, it adds a nice touch.

Reply:
Good enough for me